docs/getting-started/brokers/django.rst

```diff
@@ -37,6 +37,10 @@ configuration values.
 .. code-block:: console
 
     $ python manage.py syncdb
+    
+ Or if you are using Django 1.9 or later
+ 
+    $ python manage.py migrate kombu_transport_django
```

Comment:
As @adamchainz has already pointed out, this should be the preferred mechanism from Django 1.7 onwards. Django 1.9 just removed the `syncdb` command.